src/pages/en/guides/markdown-content.md

```diff
@@ -1,139 +1,37 @@
-        ],
-        rehypePlugins: [
-          // Add a Rehype plugin that you want to enable for your project.
-          // If you need to provide options for the plugin, you can use an array and put the options as the second item.
-          // 'rehype-slug',
-          // ['rehype-autolink-headings', { behavior: 'prepend'}],
-        ],
-      },
-    ],
-  },
-};
-```
-
-You can provide names of the plugins as well as import them:
-
-```js
-import autolinkHeadings from 'remark-autolink-headings';
-
-// astro.config.mjs
-export default {
-  markdownOptions: {
-    render: [
-      '@astrojs/markdown-remark',
-      {
-        remarkPlugins: [[autolinkHeadings, { behavior: 'prepend' }]],
-      },
-    ],
-  },
-};
-```
-
-### Syntax Highlighting
+## Markdown Pages
 
-Astro comes with built-in support for [Prism](https://prismjs.com/) and [Shiki](https://shiki.matsu.io/). By default, Prism is enabled. You can modify this behavior by updating the `@astrojs/markdown-remark` options:
+Astro treats any `.md` file inside of the `/src/pages` directory as a page. Placing a file in this directory, or any sub-directory, will automatically build a page route using the pathname of the file.
 
-```js
-// astro.config.mjs
-export default {
-  markdownOptions: {
-    render: [
-      '@astrojs/markdown-remark',
-      {
-        // Pick a syntax highlighter. Can be 'prism' (default), 'shiki' or false to disable any highlighting.
-        syntaxHighlight: 'prism',
-        // If you are using shiki, here you can define a global theme and
-        // add custom languages.
-        shikiConfig: {
-          theme: 'github-dark',
-          langs: [],
-          wrap: false,
-        },
-      },
-    ],
-  },
-};
-```
-
-You can read more about custom Shiki [themes](https://github.com/shikijs/shiki/blob/main/docs/themes.md#loading-theme) and [languages](https://github.com/shikijs/shiki/blob/main/docs/languages.md#supporting-your-own-languages-with-shiki).
+### Layouts
 
-## Markdown Pages
+Markdown pages have a special front matter property for `layout` that defines the relative path to an `.astro` [layout component](/en/core-concepts/layouts). This component will wrap your Markdown content, providing a page shell and any other page template elements. 
 
-Astro treats any `.md` files inside of the `/src/pages` directory as pages. These files can contain frontmatter, but are otherwise processed as plain markdown files and do not support components. If you're looking to embed rich components in your markdown, take a look at the [Markdown Component](#astros-markdown-component) section.
+> 🛎️ Markdown pages are children for layout components. The Markdown content is placed into the layout component's `<slot />` element, and is rendered as HTML.
 
-### Layouts
+All other front matter properties defined in your `.md` page can be passed to the layout component as properties of the `content` object prop.
```

Comment:
```suggestion
All other frontmatter properties defined in your `.md` page will be passed into the layout as properties of the `content` object prop.
```

Reply:
Interesting distinction. Are the properties "passed" if you don't specify and ask for them, or are they merely "available to be passed"? If a tree falls in a forest...

Reply:
I would accept Fuzzy's distinction: stuff is passed in to a layout from the outside, an outside that doesn't know what the layout wants. It's like I pass you a cup of tea, but you can just drop it on the floor if you don't need it. Still passed it though. 😅

Reply:
If we go with that, are we confident it should be passed "into" (vs just passed "to"). My gut still says "to" but I can't be sure that's just the language I want to hear, vs correct technical jargon.

(With slot, I think being passed "into" because it does actually GO IN somewhere, by comparison.)

Reply:
I feel like either would be fine but would personally probably also lean towards “to” (just ever so slightly).

Reply:
OK, I'm taking Fuzzy's WILL and Chris' TO. Go team!